### DISCUSSION TITLE
JESUS CHRIST HELP

### DISCUSSION THREAD

**Discussion by leondif** [hlky/stable-diffusion-webui#497] [Q&A]
When i open the public url it says that i need to enter login credentials. WHAT LOGIN CREDENTIALS
![help](https://user-images.githubusercontent.com/112673814/188049164-f82151e5-ac6c-4f16-9b24-7326f309daf3.png)

**Comment by salieri-dev** [comment]
it was a:

login: webui
password: any password you have specified with a command line parameter
